discussment-core/src/main/java/org/danekja/discussment/core/accesscontrol/service/impl/PermissionService.java

```diff
@@ -174,7 +171,7 @@ public boolean canRemovePosts(IDiscussionUser user, Discussion discussion) {
     public boolean canViewPost(IDiscussionUser user, Post post) {
         if (user == null) {
             return false;
-        } else if (Objects.equals(user.getDiscussionUserId(), post.getUserId())) {
```

Comment:
Why this change? AFAIK these are equal with the differenc that use of Objects.equals prevents NPEs in case any of the compared objects is null.